maintainers/scripts/mdize-module.sh

```diff
@@ -0,0 +1,69 @@
+#! /usr/bin/env nix-shell
+#! nix-shell -I nixpkgs=. -i bash -p delta jq perl
+
+cat <<'EOF'
+This script attempts to automatically convert option descriptions from
+DocBook syntax to markdown. Naturally this process is incomplete and
+imperfect, so any changes generated by this script MUST be reviewed.
+
+
```

Comment:
Would it make sense to add a non-exhaustive list of known issues to be aware of?